Userland/Services/WebContent/ConsoleGlobalObject.cpp

```diff
@@ -22,7 +22,7 @@ void ConsoleGlobalObject::initialize(JS::Realm& realm)
     Base::initialize(realm);
 
     // $0 magic variable
-    define_native_accessor(realm, "$0", inspected_node_getter, nullptr, 0);
+    define_native_accessor(realm, "$0", dollar_zero_getter, nullptr, 0);
```

Comment:
Either use the literal name, like here:

https://github.com/SerenityOS/serenity/blob/24caacfe28fc9f07c96157c88d87fb08045c6080/Userland/Libraries/LibJS/Contrib/Test262/GlobalObject.cpp#L21

Or describe it, like here

https://github.com/SerenityOS/serenity/blob/24caacfe28fc9f07c96157c88d87fb08045c6080/Userland/Libraries/LibJS/Runtime/RegExpConstructor.cpp#L160-L161

But not like this pls :sweat_smile:

Reply:
Ah, ok. I forgot `$` was a valid character.